src/NServiceBus.Core/Unicast/UnicastBusConfig.cs

```diff
@@ -52,6 +52,10 @@ public string DistributorDataAddress
         /// Gets/sets the address to which messages received will be forwarded.
         /// </summary>
         [ConfigurationProperty("ForwardReceivedMessagesTo", IsRequired = false)]
+        [ObsoleteEx(
+            TreatAsErrorFromVersion = "6",
+            RemoveInVersion = "7",
+            ReplacementTypeOrMember = "EndpointConfiguration.ForwardReceivedMessagesTo")]
```

Comment:
Since they now are in full control over setting it I can't come up with a use case for getting it that way?(they can just use what they passed in)